### ISSUE TITLE
chat timeout QOL improvements

### ISSUE THREAD

**Issue by domino14** [domino14/liwords#946]
- if you're typing too fast, it should not delete your chat. otherwise you have to re-type it.
- if your chat is too long, it should not delete your chat but just tell you to send a shorter message (optionally the chat box should cut off properly)

**Comment by domino14** [comment]
Should we delete the timeout for game observer chat entirely? Maybe if there's not too many people?

**Comment by Arc-of-the-Pendulum** [comment]
Yeah, so Cesar, a great new idea would be to have two ratings systems.  The ad-hoc system we already have on Woogles, leave it in place.  But in addition add an ad-hoc system where daily or weekly matches are desginated at semi-random by Woogles itself.  That would allow a disparity in matches.  Players generally tend towards wanting to play at their own level and above.  And that's fine, but as we both know, real world scrabble involves more players, especially early on in tourneys.  So, having such a random system on Woogles makes "good sense"!
